mmv1/templates/terraform/examples/vmware_engine_private_cloud_full.tf.erb

```diff
@@ -11,14 +12,22 @@ resource "google_vmwareengine_private_cloud" "<%= ctx[:primary_resource_id] %>"
     cluster_id = "<%= ctx[:vars]['management_cluster_id'] %>"
     node_type_configs {
       node_type_id = "standard-72"
-      node_count   = 3
+      node_count   = 1
       custom_core_count = 32
     }
   }
+
+  # `Type` is an immutable field from the user's side. When a PC of
+  # type TIME_LIMITED is upgraded to STANDARD, ignore changes to `type`
+  # to prevent terraform from destroying and recreating the resource.
+  lifecycle {
+    ignore_changes = [type]
```

Comment:
Is this still needed with the `ignore_read`? `ignore_read` should make it that the field value will only change when the user changes it.

Reply:
Removed `lifecycle.ignore_changes`.